rdr_service/config.py

```diff
@@ -103,6 +103,7 @@
 GENOMIC_SKIP_MISSING_FILETYPES = "genomic_skip_missing_file_types"
 GENOMIC_GC_ID_MAPPING = "gc_name_to_id_mapping"
 GENOMIC_PIPELINE_IDS = "genomic_pipeline_ids"
+GENOMIC_APPOINTMENT_GROR_EMAIL = "genomic_appointment_gror_email"
```

Comment:
Since this is a specific email for Color PM, can we change this to encompass that, maybe `GENOMIC_COLOR_PM_EMAIL`